### PR TITLE
Feature/projects list view

### LINKED ISSUE #5: Do a UI so that students can see a listing of the projects (the UI is provided with fake projects for now)

### PR DESCRIPTION
**Pull request by asfoury**
The list of projects view has been implemented and tested

### CODE REVIEW

File: app/src/main/java/com/sdp13epfl2021/projmag/adapter/ItemAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        holder.textView.text = project.name.toString()
+        // set the lab name
+        holder.labNameView.text = project.lab
+        // add the tags to the project
+       for(tag in project.tags) {
+           val chipView : Chip = Chip(context)
+           chipView.text = tag
+
+           holder.linearLayoutView.addView(chipView)
+       }
```
Comment (arnauds5) [inline app/src/main/java/com/sdp13epfl2021/projmag/adapter/ItemAdapter.kt]: When there are more than 3-4 tags, the next tags are cut in the middle.

Comment (asfoury) [reply app/src/main/java/com/sdp13epfl2021/projmag/adapter/ItemAdapter.kt]: Indeed, thanks for pointing this out and providing the fix! I updated the code.

**Review by arnauds5 — APPROVED** [review]
Looks good

**Review by lpatel99 — APPROVED** [review]
Everything looks all right to me.

**Pull request merged**